mldsa/ntt.c

```diff
@@ -7,6 +7,16 @@
 #include "params.h"
 #include "reduce.h"
 
+static int32_t mld_fqmul(int32_t a, int32_t b)
```

Comment:
Declare this in a .h file so it can called from poly.c?

Reply:
Hmmmm - no. The pre-condition that we used for fqmul does not hold in the pointwise multiplication. I'd like to keep this separate at least until we have a proof for poly_pointwise_montgomery.